libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -385,21 +385,24 @@ public static async Task<PromptRecognizerResult<TokenResponse>> RecognizeTokenAs
             }
             else if (turnContext.Activity.Type == ActivityTypes.Message)
             {
-                // regex to check if code supplied is a 6 digit numerical code (hence, a magic code).
-                var magicCodeRegex = new Regex(@"(\d{6})");
-                var matched = magicCodeRegex.Match(turnContext.Activity.Text);
-                if (matched.Success)
+                if (turnContext.Activity.Text != null)

```

Comment:
Should this be ?
```cs
if (!string.IsNullOrEmpty(turnContext.Activity.Text))
```

Reply:
yes! guh, been a while since I made a PR in dotnet. Will update :)

Reply:
Updated! Thanks, Eric